### PR TITLE
Correctly calculate release diff

### PR DESCRIPTION
**Pull request by mouhsen-ibrahim**

### CODE REVIEW

File: services/frontend-service/src/ui/components/ServiceLane/ServiceLane.tsx [hunk 1]
```diff
@@ -14,20 +14,38 @@ You should have received a copy of the GNU General Public License
...
 Copyright 2021 freiheit.com*/
-import { useDeployedReleases } from '../../utils/store';
+import { useDeployedReleases, useReleases } from '../../utils/store';
 import { ReleaseCard } from '../ReleaseCard/ReleaseCard';
 import { Button } from '../button';
 import { DeleteWhite, HistoryWhite } from '../../../images';
-import { Application } from '../../../api/api';
+import { Application, Release } from '../../../api/api';
+
+function getReleaseDiff(releases: Array<Release>, version1: number, version2: number) {
```
Comment (sven-urbanski-freiheit-com) [inline services/frontend-service/src/ui/components/ServiceLane/ServiceLane.tsx:23]:
```suggestion
function getNumberOfReleasesBetween(releases: Array<Release>, version1: number, version2: number) {
```
suggestion to rename function

Comment (sven-urbanski-freiheit-com) [reply services/frontend-service/src/ui/components/ServiceLane/ServiceLane.tsx:23]: I would also rename "version1" to "lowerVersion" and 2 to "higherVersion" or something like that

Comment (mouhsen-ibrahim) [reply services/frontend-service/src/ui/components/ServiceLane/ServiceLane.tsx]: done

File: services/frontend-service/src/ui/components/ServiceLane/ServiceLane.test.tsx [hunk 2]
```diff
@@ -94,9 +94,56 @@ describe('Service Lane', () => {
...
+        diff: '0',
+        releases: [
+            {
+                version: 1,
+                sourceMessage: 'test1',
+                sourceAuhor: 'test',
+                sourceCommitId: 'commit1',
+                createdAt: new Date(2002),
+            },
+        ] as any,
```
Comment (sven-urbanski-freiheit-com) [inline services/frontend-service/src/ui/components/ServiceLane/ServiceLane.test.tsx:108]: why `as any` and not `as Release[]`?

File: services/frontend-service/src/ui/utils/store.tsx [hunk 3]
```diff
@@ -368,3 +368,11 @@ export const useReleaseInfo = (app: string, version: number) =>
         }
         return releaseInfo;
     });
+
+// Get all releases for an app
+export const useReleases = (app: string) =>
```
Comment (sven-urbanski-freiheit-com) [inline services/frontend-service/src/ui/utils/store.tsx:373]:
```suggestion
export const useReleasesForApp = (app: string) =>
```

**Review by sven-urbanski-freiheit-com — APPROVED** [review]

File: services/frontend-service/src/ui/components/ServiceLane/ServiceLane.test.tsx (hunk 2 above)
Comment (mouhsen-ibrahim) [reply services/frontend-service/src/ui/components/ServiceLane/ServiceLane.test.tsx:108]: I copied it like this from another code, I'll fix t now

**Pull request merged**